python/cucim/tests/performance/clara/test_read_region_memory_usage.py

```diff
@@ -40,8 +40,10 @@ def test_read_region_cuda_memleak(testimg_tiff_stripe_4096x4096_256_jpeg):
 
     # The difference in memory usage should be less than 180MB.
     # Note: Since we cannot measure GPU memory usage for a process,
-    #       we use a rough number (experimentally measured).
-    assert mem_usage_history[4] - mem_usage_history[1] < 180.0
+    #       we use a rough number.
+    #       (experimentally measured, assuming that each image load
+    #        consumes around 50MB of GPU memory).
+    assert mem_usage_history[5] - mem_usage_history[9] < 180.0
```

Comment:
Were we wanting to increase the `180.0` limit as well?

Reply:
Hi @jakirkham ,

Since each method call is expected to increase ~50MB of GPU memory, it is possible that we may miss the actual memory leak if the value is more than 200MB (50MB x 4).

For this reason, instead of increasing the value, this chooses the 6th and 10th samples, instead of the 2nd and 5th samples for comparison.

Reply:
Ok let's give this a try then and see how it goes